### ISSUE TITLE
JSON Spec for sources[] and sources[][]

### ISSUE THREAD

**Issue by tischi** [mobie/mobie-viewer-fiji#620]
Agree.
My problem with this is that at the parent level `sourceTransforms` there is already a spec for `affine` that contains transformation information as a nested object.
I also find it extremely confusing that different `sourceTransforms` entries (`affine` or `crop` vs. `grid` or `transformedGrid`)  have different specs for their `sources`. (List vs. List of Lists) 
Using dedicated key names with own specs would make everything more clear.
I think we should try to keep things as consistent as possible in the view spec.

_Originally posted by @martinschorb in https://github.com/mobie/mobie-viewer-fiji/issues/619#issuecomment-1049870930_

**Comment by tischi** [comment]
I agree, it would be nice to have two different names here.

**Comment by constantinpape** [comment]
Yeah I agree, but this is a bigger operation and I don't have time to work on it on the python side now.
